Set correct che api external enviroment variable. Use official released workspace client.

AndrienkoAleksandr opened the pull request:
Set correct che api external env. variable, because pull request: https://github.com/eclipse/che/pull/9475 was merged.
Use official released Eclipse CHE workspace client. 
Set skip lib checks to prevent fail build.

AndrienkoAleksandr committed:
Fix CHE API env variables name.

AndrienkoAleksandr committed:
Use latest released version of the workspace library

Signed-off-by: Oleksandr Andriienko <oandriie@redhat.com>

benoitf commented on theia-machines-extension/tsconfig.json:
```diff
@@ -1,5 +1,6 @@
 {
     "compilerOptions": {
+        "skipLibCheck": true,
```
why do we need to skip the check ?

AndrienkoAleksandr replied:
Without skip check lib build fails.

AndrienkoAleksandr committed:
Use Theia 0.3.10. Update workspace-client version. Don't use skipLibCheck, use strict js.

Signed-off-by: Oleksandr Andriienko <oandriie@redhat.com>

AndrienkoAleksandr committed:
Important fixes:
Apply yarn.lock for reproducible build.
Fix incorrect list machines when user open machine panel and do browser page refresh.
Synchronize getting workspaceId and che api end point.
Code clean up.
Update version of @eclipse-che/workspace-client

Signed-off-by: Oleksandr Andriienko <oandriie@redhat.com>

AndrienkoAleksandr committed:
Fix up.

Signed-off-by: Oleksandr Andriienko <oandriie@redhat.com>

AndrienkoAleksandr committed:
Fix up.

Signed-off-by: Oleksandr Andriienko <oandriie@redhat.com>

AndrienkoAleksandr commented:
@olexii4 take a look again, please. 
I applied yarn.lock file, 
fixed synchronization bug when plugin can get workspaceId or api end point earlier than it was initialized, fixed bug when list machines contains duplication when user open machine panel and refresh browser,
used Env variable service from theia-core.
